src/main/java/htsjdk/samtools/cram/build/ContainerFactory.java

```diff
@@ -52,57 +54,75 @@ public ContainerFactory(final SAMFileHeader samFileHeader, final int recordsPerS
         this.recordsPerSlice = recordsPerSlice;
     }
 
-    public Container buildContainer(final List<CramCompressionRecord> records) {
```

Comment:
Merged these two methods.  `substitutionMatrix` was never set so it's just `null` below

Reply:
You're right.  I'll add a TODO.